### ISSUE TITLE
session namespaces broken on R 3.3.0?

### ISSUE THREAD

**Issue by cmhh** [jeroenooms/opencpu#183]
Hi

I was trying to use the ```sessionid::name``` functionality and it wasn't working.  When trying to use an object in a temporary session I get the following error:

```
cmhh@debian:~$ curl http://debian/ocpu/library/stats/R/sd/json -d x=x01ec9e9bdf
comparison (1) is possible only for atomic and list types

In call:
ns == loadedNamespaces()
```

This was in R 3.3.  Going back to 3.2.5 seems to get rid of the error.

Cheers

Chris

**Comment by jeroenooms** [comment]
Hmm that sucks. I'll debug this asap. Thanks for catching this.

**Comment by jeroenooms** [comment]
Looks like this is a [bug](https://stat.ethz.ch/pipermail/r-devel/2016-May/072684.html) in R 3.3. Luckily there is a simple workaround: https://github.com/jeroenooms/opencpu/commit/622b54495424c9532f48e11e76acc5beb4324f7f.

**Comment by jeroenooms** [comment]
This is now fixed in opencpu 1.6 on cran and launchpad. Thanks again for reporting this, really helpful.

**Comment by cmhh** [comment]
Thanks for sorting it so quickly. I had actually installed opencpu-server v1.6 from github this morning to get the fix. 
Much appreciated.
On 11/05/2016 10:44 AM, Jeroen Ooms <notifications@github.com> wrote:This is now fixed in opencpu 1.6 on cran and launchpad. Thanks again for reporting this, really helpful.
